lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3953,7 +3823,7 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
   SILFunction &getPullback() const { return *vjpEmitter.pullback; }
   SILDifferentiableAttr *getAttr() const { return vjpEmitter.attr; }
   DifferentiationInvoker getInvoker() const { return vjpEmitter.invoker; }
-  PullbackInfo &getPullbackInfo() { return vjpEmitter.pullbackInfo; }
+  LinearMapInfo &getLinearMapInfo() { return vjpEmitter.linearMapInfo; }
```

Comment:
Don't rename "PullbackInfo", which is more accurate.
```suggestion
  LinearMapInfo &getPullbackInfo() { return vjpEmitter.linearMapInfo; }
```